package-lock.json

```diff
@@ -0,0 +1,1269 @@
+{
+  "name": "YarpJS",
+  "version": "1.0.0",
```

Comment:
Is this correct?

Reply:
As far as I understood, this version really matters when we publish YarpJS to the **npm** package manager [registry](https://docs.npmjs.com/misc/registry) which is the public database of JavaScript packages managed by npm (https://docs.npmjs.com/about-npm).

Thanks for highlighting this @traversaro , we should update this to 1.5.0 as soon as we publish the new tag, and before we merge the PR.

Reply:
Or should we tag the upgraded code v2.0.0, considering that the Node.js version significantly changed from 4.2.2 to 14.17.0?

Reply:
As discussed offline, changed the tag to 1.5.0 in the `package.json`, and we'll tag the repo to v2.0.0 after shaking down a bit more the upgraded `yarp.js` module.